README.md

```diff
@@ -31,8 +32,12 @@ eval "$(_SUBSTRA_COMPLETE=source_zsh substra)"
 
 From this point onwards, substra command line interface will have autocompletion enabled.
 
-See the [local install guide](./docs/local_install.md) if you need to run a full instance of the Substra platform
-locally.
+## Running the Substra platform locally
+
+You can run the Substra platform locally on your machine using using of the two following methods:
+
+* [Using docker-compose](./docs/local_install_docker_compose.md)
+* [Using kubernetes and skaffold](./docs/local_install_skaffold.md)
```

Comment:
We could recommend one the methods (skaffold?).

Reply:
Definitely :)